### DISCUSSION TITLE
Other Harvests are not shown on "Your Harvester Network"

### DISCUSSION THREAD

**Discussion by DIlbert25** [Chia-Network/chia-blockchain#4982] [Support]
Hi,

I have a Full Node on one computer (running Windows 10 and the GUI) and I have two other harvesters computers, running on WindowsPowerShell, only the harvester ( chia start harvester -r) .

The logs show that they are talking to Node, but why are they  not shown on to the "Full Node - Harvester Network"?

**Comment by DIlbert25** [comment]
Solved.

This tip could be in the FAQ ....

Another post:

"
@frangere
frangere
12 days ago

re-login by click Keys->Choose your key and then you can find your harvester machine at "Your Harvester Network". Right now, I can find my harvester machine."

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
